packages/app-store/ee/routing_forms/package.json

```diff
@@ -5,10 +5,6 @@
   "version": "0.0.0",
   "main": "./index.ts",
   "description": "It would allow a booker to connect with the right person or choose the right event, faster. It would work by taking inputs from the booker and using that data to route to the correct booker/event as configured by Cal user  ",
-  "scripts": {
-    "app-e2e": "yarn playwright test --config=playwright/config/playwright.config.ts",
-    "app-e2e-quick": "QUICK=true yarn app-e2e"
-  },
```

Comment:
Apps don't need to setup their own runners anymore. You just need to add a `*.e2e.ts` file and playwright should pick it up.